src/PrintFeature.h

```diff
@@ -16,7 +16,8 @@ enum class PrintFeatureType: unsigned char
     SupportInfill,
     MoveCombing,
     MoveRetraction,
-    SupportInterface
+    SupportInterface,
+    NumPrintFeatureTypes
```

Comment:
Please put a little comment here to explain what this value is used for.

You can explicitly set the type of underlying integer to use for the enum. I'm unsure, but maybe you should also explicitly set the values of the enum constants like`SupportInterface = 6,`.